### PR TITLE
PT-20 - feat(RN:Drawer): initial component

### PR DESCRIPTION
**Pull request by cameronolivier**
## Scope: Initial RN drawer

See for previous comments: PR[#4](https://github.com/Nona-Creative/nona-playground/pull/4)

### CODE REVIEW

File: packages/mob-ui-react-native/package.json [hunk 1]
```diff
@@ -63,10 +67,12 @@
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "jest": "^25.1.0",
+    "jest-each": "^25.1.0",
```
Comment (neilrussell6) [inline packages/mob-ui-react-native/package.json:70]: traitor :D

File: packages/mob-ui-react-native/src/common/components/Drawer/Drawer.styles.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { StyleSheet } from 'react-native'
```
Comment (neilrussell6) [inline packages/mob-ui-react-native/src/common/components/Drawer/Drawer.styles.js:1]: add newline

File: packages/mob-ui-react-native/src/common/components/Drawer/Drawer.utils.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { __, divide, includes, multiply, pipe, when } from 'ramda'
+
+export const getPercentOfValue = (value) => pipe(
```
Comment (neilrussell6) [inline packages/mob-ui-react-native/src/common/components/Drawer/Drawer.utils.js:3]: fix tabbing here please

File: packages/mob-ui-react-native/package.json (hunk 1 above)
Comment (cameronolivier) [reply packages/mob-ui-react-native/package.json:70]: hahaha

Comment (cameronolivier) [reply packages/mob-ui-react-native/package.json:70]: ... I actually think that's unnecessary - I think it's already wrapped into jest.

**Review by ian-cawood — APPROVED** [review]

**Pull request merged**